pkg/render/testutils/expected_policies/dashboards.json

```diff
@@ -8,7 +8,7 @@
   "spec": {
     "order": 1,
     "tier": "allow-tigera",
-    "selector": "job-name == 'dashboards-installer'",
+    "selector": "job-name contains 'dashboards-installer'",
```

Comment:
I think we should be using a different rather than the contains operator here. There's no reason we need to use the `job-name` label AFAIK - we could have `role: dashboard-installer` applied regardless of the actual name, for example. 

My concern is just that `contains` is a pretty unusual operation, and given there's no concrete _need_ for it here it makes this policy less intuitive.